### ISSUE TITLE
Reason for stdin(stdio::null)?

### ISSUE THREAD

**Issue by whatmatrix** [Kobzol/cargo-pgo#28]
I have a command-line app that I wishes to use PGO and BOLT. It takes a few inputs from stdin at the startup. It works well with `cargo run`.

`cargo pgo` seems to be [overriding `stdin` by passing `stdio::null()`](https://github.com/Kobzol/cargo-pgo/blob/68c5d8929754c09080861f20ade3de7718995d8b/src/build.rs#L81).

Is it necessary for `cargo pgo` functionalities? `cargo pgo run` obviously works differently than `cargo run` due to this.

**Comment by Kobzol** [comment]
Hi, good point. I think that I did this only out of habit, but for `cargo pgo run` it doesn't make sense. Can you please try if [this PR](https://github.com/Kobzol/cargo-pgo/pull/29) fixes your problem?

```console
$ cargo install --git https://github.com/kobzol/cargo-pgo --branch pass-stdin
```

**Comment by Kobzol** [comment]
No problem! I'm glad that it helped. I will release a new version with the patch soon if there won't be any other changes in the meantime.

**Comment by Kobzol** [comment]
Published `0.2.4` with this change.
